README.md

```diff
@@ -6,18 +6,41 @@ Very simple, no JS necessary. Using [Hugo](https://gohugo.io/), a fast static we
 
 ## Getting started
 
-[Install Hugo](https://gohugo.io/getting-started/installing/):
+**[Install Hugo](https://gohugo.io/getting-started/installing/):**
 
 * On OSX with [homebrew](https://brew.sh/): `brew install hugo`
 * Or download a release from here: https://github.com/gohugoio/hugo/releases
 
+**[Install nodeJS](https://nodejs.org/en/download/)**
```

Comment:
What does a user need nodeJS for? I think we should put this a bit later, and first show how to run hugo plain, as before: `hugo server`.

Then explain what Node.js is needed for, and how to install it